src/main/resources/META-INF/accesstransformer.cfg

```diff
@@ -126,6 +127,9 @@ public net.minecraft.client.renderer.entity.ItemRenderer m_115162_(Lcom/mojang/b
 public net.minecraft.client.renderer.entity.ItemRenderer m_115189_(Lnet/minecraft/client/resources/model/BakedModel;Lnet/minecraft/world/item/ItemStack;IILcom/mojang/blaze3d/vertex/PoseStack;Lcom/mojang/blaze3d/vertex/VertexConsumer;)V # renderModelLists
 public net.minecraft.client.renderer.entity.LivingEntityRenderer m_115326_(Lnet/minecraft/client/renderer/entity/layers/RenderLayer;)Z # addLayer
 public net.minecraft.client.renderer.texture.TextureAtlasSprite m_118415_()I # getFrameCount
+public net.minecraft.client.renderer.chunk.ChunkRenderDispatcher$CompiledChunk f_112749_ # hasBlocks
+public net.minecraft.client.renderer.chunk.ChunkRenderDispatcher$CompiledChunk f_112750_ # hasLayer
+public net.minecraft.client.renderer.chunk.ChunkRenderDispatcher$CompiledChunk f_112751_ # isCompletelyEmpty
```

Comment:
Please run the `checkATs` task to automatically move these lines to their proper sorted position.